chart/templates/mayastor/csi/csi-node-daemonset.yaml

```diff
@@ -78,7 +78,10 @@ spec:
         - "--csi-socket={{ .Values.csi.node.pluginMounthPath }}/{{ .Values.csi.node.socketPath }}"
         - "--node-name=$(MY_NODE_NAME)"
         - "--grpc-endpoint=$(MY_POD_IP):10199"{{ if .Values.csi.node.nvme.io_timeout }}
-        - "--nvme-core-io-timeout={{ .Values.csi.node.nvme.io_timeout }}"{{ end }}{{ if .Values.csi.node.nvme.ctrl_loss_tmo }}
+        - "--nvme-io-timeout={{ .Values.csi.node.nvme.io_timeout }}"
+        - "--nvme-core-io-timeout={{ .Values.csi.node.nvme.io_timeout }}"{{ else }}
+        - "--nvme-core-io-timeout={{ .Values.io_engine.nvme.ioTimeout }}10s"
```

Comment:
Seems duplicated?

Reply:
It's not, but the order is swapped in the else..
I'll keep it the same order